### PR TITLE
Add sport monks data provider

### PR DESCRIPTION
**Pull request by dmitrydao**

### CODE REVIEW

File: docs/Data Provider Nodes/sport-monks-oracle.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+permalink: "docs/sport-monks-oracle/"
+---
+
+SportMonks is a provider of reliable sport data feeds, powered by a fast, highly available, and easy to implement API. They currently provide live data on three sports: soccer, F1, and cricket. They offer a variety of data such as live scores, player statistics, team stats, seasons stats, bookmakers and markets, (live) odds, predictions and coverage across more than 1,200 leagues. This oracle will initially provide Match Results and Toss Results for the 2021 Indian Premier League Cricket season.
+
+# Steps For Using This Oracle
+
+- Write and deploy your [Chainlink](../intermediates-tutorial/) contract using the network details below
+- Fund it with [LINK](../link-token-contracts/)
+- Call your [request method](#section-chainlink-examples)
```
Comment (alexroan) [inline docs/Data Provider Nodes/sport-monks-oracle.md:15]: Anchor broken, and not sure if this anchor will work after #54 is merged?

File: docs/Data Provider Nodes/sport-monks-oracle.md [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+  req.add("season_id", _seasonId);
+  sendChainlinkRequestTo(_oracle, req, oraclePayment);
+}
+```
+### `fulfill` function
+
+```solidity
+bytes32 public data;
+
+function fulfill(bytes32 _requestId, bytes32 _data_)
```
Comment (alexroan) [inline docs/Data Provider Nodes/sport-monks-oracle.md:150]: `_data_` ?

**Review by alexroan — CHANGES_REQUESTED** [review]

File: docs/Data Provider Nodes/sport-monks-oracle.md (hunk 2 above)
Comment (dmitrydao) [reply docs/Data Provider Nodes/sport-monks-oracle.md]: thanks for catching

File: docs/Data Provider Nodes/sport-monks-oracle.md (hunk 1 above)
Comment (dmitrydao) [reply docs/Data Provider Nodes/sport-monks-oracle.md:15]: doesn't but we'll find a way to get anchors back

Comment (alexroan) [reply docs/Data Provider Nodes/sport-monks-oracle.md:15]: Can we at least get this one to work? I noticed that the adapters seem to work fine.

**Review by alexroan — APPROVED** [review]

**Pull request merged**